runtime/onert/core/include/compiler/LoweredGraph.h

```diff
@@ -54,11 +54,6 @@ class LoweredGraph
   }
 
 private:
-  void makeOperationLowerInfo(
-    ir::OperandIndexMap<std::unique_ptr<compiler::OperandLowerInfo>> &operands_lower_info,
-    const compiler::BackendResolver &backend_resolver);
-  void manipulateLowerInfo(
-    ir::OperandIndexMap<std::unique_ptr<compiler::OperandLowerInfo>> &operands_lower_info);
```

Comment:
Doesn't it make SAM score down ?

Reply:
Well, then I will create a method for `LoweredGraph.cc:90-147`. But this time it is going to be just one, no need to be two 😃